src/Psalm/Internal/Analyzer/Statements/Expression/Fetch/ConstFetchAnalyzer.php

```diff
@@ -155,7 +155,7 @@ public static function getGlobalConstType(
                 case 'PHP_PREFIX':
                 case 'PHP_SAPI':
                 case 'PHP_SYSCONFDIR':
-                    return Type::getString();
```

Comment:
Not all of those are necessarily non-empty: https://3v4l.org/oSTv2

Reply:
I would suggest to limit the change to those constants you had in your original issue.

Reply:
I found 3 constants possibly empty and fixed them.